marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/saveSavedQuery.sjs

```diff
@@ -43,7 +43,15 @@ if (queryDocument.savedQuery.propertiesToDisplay == null || queryDocument.savedQ
     ds.throwBadRequest("Entity type properties to be displayed cannot be empty");
 }
 
-let id = queryDocument.savedQuery.id;
+const id = queryDocument.savedQuery.id;
+const positiveQuery = cts.andQuery([cts.collectionQuery("http://marklogic.com/data-hub/saved-query"), cts.jsonPropertyValueQuery("name", queryDocument.savedQuery.name),
```

Comment:
A general technique for making queries easier to read is to put each query on a separate line - e.g.

```
cts.andQuery([
  cts.collectionQuery(".."),
  cts.jsonPropertyValueQuery(...),
  cts.jsonPropertyValueQuery(...)
])
```